module/VuFind/src/VuFind/Config/Locator.php

```diff
@@ -38,75 +38,42 @@
  */
 class Locator
 {
-    /**
-     * Mode for getConfigPath: try to find a local file but fall back to base file
-     * if not available.
-     *
-     * @var int
-     */
-    public const MODE_AUTO = 0;
-
-    /**
-     * Mode for getConfigPath: try to find a local file.
-     *
-     * @var int
-     */
-    public const MODE_LOCAL = 1;
-
-    /**
-     * Mode for getConfigPath: get the base configuration file path.
-     *
-     * @var int
-     */
-    public const MODE_BASE = 2;
-
-    /**
-     * Default configuration path.
-     *
-     * @var string
-     */
-    public const DEFAULT_CONFIG_PATH = 'config/vufind';
-
     /**
      * Get the file path to the local configuration file (null if none found).
      *
      * @param string $filename config file name
-     * @param string $path     path relative to VuFind base (optional; defaults
-     * to config/vufind if set to null)
+     * @param string $path     path relative to VuFind base (optional; use null for
+     * default)
      * @param bool   $force    force method to return path even if file does not
      * exist (default = false, do not force)
      *
      * @return string
+     *
+     * @deprecated Use PathResolver instead
      */
     public static function getLocalConfigPath(
         $filename,
         $path = null,
         $force = false
     ) {
-        if (null === $path) {
-            $path = 'config/vufind';
-        }
-        if (defined('LOCAL_OVERRIDE_DIR') && strlen(trim(LOCAL_OVERRIDE_DIR)) > 0) {
-            $path = LOCAL_OVERRIDE_DIR . '/' . $path . '/' . $filename;
-            if ($force || file_exists($path)) {
-                return $path;
-            }
-        }
-        return null;
+        return static::getPathResolver()
+            ->getLocalConfigPath($filename, $path, $force);
     }
 
     /**
      * Get the file path to the base configuration file.
      *
      * @param string $filename config file name
-     * @param string $path     path relative to VuFind base (optional; defaults
-     * to config/vufind
+     * @param string $path     path relative to VuFind base (optional; use null for
```

Comment:
This should be a ?string now (as should the one in getLocalConfigPath, though that's been wrong for a while).